main.py

```diff
@@ -0,0 +1,63 @@
+# Task 1: Simple Quiz implementaion
+import json
+ 
+score = 0
+
+# Functions prototype
+def check_answer(entered_answer, correct_answer):
+    global score
+    if int(entered_answer) == correct_answer:        
+        score += 1  # Increase score for a correct answer
+        return "correct"
+       
+    else:
+        return "wrong"
```

Comment:
lets compare strings, and make the function returm bool: (Troe or False) and increase the score in the main.
this way we get rid of the `global score`, and make it simpler.